src/aat/resources/features/F-026/F-026.feature

```diff
@@ -0,0 +1,58 @@
+@F-026
+Feature: F-026: Get a case data with UI layout
```

Comment:
Feature title is wrong. With an 'a' inserted, it reads like an operation that returns just 1 case data. Starting from this title with just a single letter difference, the rest of the files became much more confusing to understand. I initially thought, like, if this is to get 'a' case data, where is the case id for it in the test dat files, etc.

Reply:
Above comment preserved, the original title will be made more clear and self descriptive after a total review of original swagger documentation.

Reply:
Corrected this typo and copy paste issue.
Feature: F-026: Get case data with UI layout